### PR TITLE
Prepare Text Analytics 5.2.0 release

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.2.0-beta.5 (Unreleased)
+## 5.2.0 (2022-09-08)
```
Comment (deyaaeldeen) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:3]: I think the empty sections should be removed too.

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 2]
```diff
@@ -555,16 +587,17 @@ await foreach (AnalyzeHealthcareEntitiesResultCollection documentsInPage in heal
...
-- Custom Multi Category Classification (see sample [here][multi_category_classify_sample])
+
+* Named Entities Recognition
+* PII Entities Recognition
+* Linked Entity Recognition
+* Key Phrase Extraction
+* Sentiment Analysis
+* Custom Named Entity Recognition (see sample [here][recognize_custom_entities_sample])
+* Custom Single Category Classification (see sample [here][single_category_classify_sample])
+* Custom Multi Category Classification (see sample [here][multi_category_classify_sample])
```
Comment (deyaaeldeen) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:601]: Text Analytics for health is not here

**Review by deyaaeldeen — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 3]
```diff
@@ -29,24 +31,27 @@ This table shows the relationship between SDK versions and supported API version
 
 |SDK version|Supported API version of service
 |-|- |
+|5.2.0 | 3.0, 3.1, 2022-05-01 (default)
 |5.2.0-beta.4 | 3.0, 3.1, 2022-05-01 (default)
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:35]: do we need to remove these betas.

**Review by jsquire — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 4]
```diff
@@ -707,24 +742,25 @@ To learn more about other logging mechanisms see [here][logging].
...
-- [Custom Multi Category Classification][multi_category_classify_sample]
+* [Detect Language][detect_language_sample]
+* [Analyze Sentiment][analyze_sentiment_sample]
+* [Extract Key Phrases][extract_key_phrases_sample]
+* [Recognize Named Entities][recognize_entities_sample]
+* [Recognize PII Entities][recognize_pii_entities_sample]
+* [Recognize Linked Entities][recognize_linked_entities_sample]
+* [Recognize Healthcare Entities][analyze_healthcare_sample]
+* [Custom Named Entity Recognition][recognize_custom_entities_sample]
+* [Custom Single Category Classification][single_category_classify_sample]
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:753]: Custom Single **Label** Classification and Custom Multi **Label** Classification

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 5]
```diff
@@ -707,24 +742,25 @@ To learn more about other logging mechanisms see [here][logging].
...
-- [Custom Single Category Classification][single_category_classify_sample]
-- [Custom Multi Category Classification][multi_category_classify_sample]
+* [Detect Language][detect_language_sample]
+* [Analyze Sentiment][analyze_sentiment_sample]
+* [Extract Key Phrases][extract_key_phrases_sample]
+* [Recognize Named Entities][recognize_entities_sample]
+* [Recognize PII Entities][recognize_pii_entities_sample]
+* [Recognize Linked Entities][recognize_linked_entities_sample]
+* [Recognize Healthcare Entities][analyze_healthcare_sample]
+* [Custom Named Entity Recognition][recognize_custom_entities_sample]
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md:752]: entities?

**Review by mssfang — APPROVED** [review]
small corrections and looks good.

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 3 above)
Comment (heaths) [reply sdk/textanalytics/Azure.AI.TextAnalytics/README.md:35]: Probably a good idea. Apart from the CHANGELOG, we don't make mention elsewhere of betas.

**Pull request merged**